world-client/src/state/player.js

```diff
@@ -12,6 +14,8 @@ export default class Player {
 
     this.character.setBounce(0, 0);
     this.character.setCollideWorldBounds(true);
+    this.character.body.onOverlap = true;
```

Comment:
Tole sklepam da je Phaser internals in ni nič na nas? onOverlap mi vleče bolj na event kot pa da prejme "true".